workshops/management/commands/check_for_workshop_websites_updates.py

```diff
@@ -228,8 +242,21 @@ def handle(self, *args, **options):
                         events_for_update[event.slug] = changes
                         print('Detected changes in {}'.format(event.slug))
 
+            except GithubException:
+                print('GitHub error when accessing {} repo'.format(event.slug),
+                      file=sys.stderr)
+
+            except socket.timeout:
+                print('Timeout when accessing {} repo'.format(event.slug),
+                      file=sys.stderr)
+
             except WrongWorkshopURL:
-                print('Wrong URL for {}'.format(event.slug))
+                print('Wrong URL for {}'.format(event.slug), file=sys.stderr)
 
             except requests.exceptions.RequestException:
-                print('Network error when accessing {}'.format(event.slug))
+                print('Network error when accessing {}'.format(event.slug),
+                      file=sys.stderr)
+
+            except Exception as e:
```

Comment:
This is equivalent to the "else: fail(...)" discussion we're having - you know what exceptions ought to be able to come out, but you catch 'Exception' in case something new arises.